src/odb_loose.c

```diff
@@ -121,8 +121,9 @@ static size_t get_binary_object_header(obj_hdr *hdr, git_buf *obj)
 }
 
 static int parse_header(
-	obj_hdr *out, size_t *out_len, unsigned char *data, size_t data_len)
+	obj_hdr *out, size_t *out_len, unsigned char *_data, size_t data_len)
 {
+	const char *data = (char *)_data;
```

Comment:
Looks like this chunk belongs into "odb: support large loose objects"? You should also make the data parameter const.

Reply:
Fixed.